### PR TITLE
zombie event overhaul

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR 
figured the old event was a bit shit. now it just uses the gamerule instead.
also made zombie infection rates a bit better.

**Screenshots**

**Changelog**

:cl:
- add: The zombie midround event now just starts zombie mode.
- tweak: Zombie melee infection rates have been lowered.

### CODE REVIEW

File: Resources/Prototypes/GameRules/events.yml [hunk 1]
```diff
@@ -158,7 +158,7 @@
   id: ZombieOutbreak
   config:
     !type:StationEventRuleConfiguration
-    id: ZombieOutbreak
+    id: Zombie
```
Comment (mirrorcult) [inline Resources/Prototypes/GameRules/events.yml:161]: this is supposed to be the same as the gamerule ID

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/GameRules/events.yml (hunk 1 above)
Comment (mirrorcult) [reply Resources/Prototypes/GameRules/events.yml:161]: ..or, okay, this is weirder, i thought this handled both. this isnt really how its intended for this to work, but I guess it works

Comment (moonheart08) [reply Resources/Prototypes/GameRules/events.yml:161]: > ..or, okay, this is weirder, i thought this handled both. this isnt really how its intended for this to work, but I guess it works

This was something i was aware of when adding event rule configs and it's intended
and no, it's not *great*, but it beats refactoring every rule to support  running multiple times at once

Comment (EmoGarbage404) [reply Resources/Prototypes/GameRules/events.yml:161]: Should i just leave it for now?

**Review by ShadowCommander — APPROVED** [review]
Looks fine as long as the other review is resolved.

**Pull request merged**